python/ray/includes/absl.pxd

```diff
@@ -0,0 +1,74 @@
+from libcpp.utility cimport pair
+
+cdef extern from "absl/container/flat_hash_map.h" namespace "absl" nogil:
```

Comment:
How do we ensure this is correct?
seems to me a bunch of functions are missing here, such as `merge` and `contains`